src/stories/Wrapper.tsx

```diff
@@ -410,6 +410,12 @@ const Wrapper: Decorator = (Story) => {
       externalLinkOpenRequested={(url) => {
         alert(`External link "${url}" requested!`);
       }}
+      // avatarUrlOverride={(user) => {
+      //   if (user.id === "132819036282159104")
+      //     return "https://cdn.discordapp.com/emojis/698964060770926684.png";
+      //
+      //   return null;
+      // }}
```

Comment:
is this supposed to be here?

Reply:
Yes, this is for "documentation" purposes. That entire file also serves as a "how do I use this" doc before I write actual documentation. This is an example usage